pack/commands.cfg

```diff
@@ -0,0 +1,25 @@
+# -----------------------------------------------------------------
+#
+#      OpenStack standard check
+#      Need  https://github.com/kedark3/check-osp
+#
+# -----------------------------------------------------------------
+
+
+## Check commands for openstack
+define command{
+       command_name     check_vm_count
```

Comment:
For these commands I would distinguish them from the RHV checks, so we know they are specific to OSP